src/game.cpp

```diff
@@ -3619,10 +3625,13 @@ void game::draw_sidebar()
     if( sideStyle ) {
         werase( w_status2 );
     }
-    u.disp_status( w_status, w_status2 );
 
+    // sidestyle ? narrow_1 : wider_0
```

Comment:
Please remove the commented section.

Reply:
why ? am i not allowed to leave comment  related to the code ?

Reply:
Oh, it is related to the code? Then leave it, I just thought it is a left-over.